core/include/cubos/core/ecs/system.hpp

```diff
@@ -136,16 +135,16 @@ namespace cubos::core::ecs
             /// @param world The world to query from.
             /// @param commands The commands object.
             /// @returns The requested query data.
-            static Type fetch(const World& world, Commands& commands);
+            static Type fetch(World& world, Commands& commands);
 
             /// @param fetched The fetched query.
             /// @returns The query.
-            static Type arg(const World& world, Type&& fetched);
+            static Type arg(Type&& fetched);
         };
 
-        template <> struct SystemFetcher<Debug>
+        template <> struct SystemFetcher<World&>
         {
-            using Type = Debug;
+            using Type = World*;
```

Comment:
Is `Type` still used?

Reply:
Yeah, its used in the fetcher for tuples to determine its `Type`.
```cpp
        template <typename... Args> struct SystemFetcher<std::tuple<Args...>>
        {
            using Type = std::tuple<typename SystemFetcher<Args>::Type...>;
            ...
```